src/events/mod.rs

```diff
@@ -901,11 +897,13 @@ impl<'a> BytesCData<'a> {
     /// | `&`       | `&amp;`
     /// | `'`       | `&apos;`
     /// | `"`       | `&quot;`
-    pub fn escape(self) -> BytesText<'a> {
-        BytesText::from_escaped(match escape(&self.content) {
+    pub fn escape(self, decoder: Decoder) -> Result<BytesText<'a>> {
```

Comment:
Because `BytesText::escape` and `BytesText::partial_escape` introduced only in new 0.24 release, the changes in their signatures are not reflected in the changelog. Moreover, the next PRs will change that signature back

Reply:
+1